projects/ngx-meta/src/routing/src/metadata-route-strategy.ts

```diff
@@ -1,10 +1,12 @@
 import { InjectionToken } from '@angular/core'
 import { ActivatedRouteSnapshot } from '@angular/router'
+import { MetadataValues } from '@davidlj95/ngx-meta/core'
 
-export type MetadataRouteStrategy = <T>(
```

Comment:
Doesn't make sense to specify type here, given the strategy should load some object from route, without caring what the object contains. It's not its responsibility